RentACarWebAPI/Startup.cs

```diff
@@ -26,24 +24,21 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddDbContext<RentACarDbContext>(options =>
+                options.UseSqlServer(Configuration.GetConnectionString("Database")
+            ));
```

Comment:
minor: move one of the braces to the line it belongs to close the method call:
```csharp
            services.AddDbContext<RentACarDbContext>(options =>
                options.UseSqlServer(Configuration.GetConnectionString("Database"))
            );
```

Reply:
All right! Thanks!